autolens/plot/plotters/fit_imaging_plotters.py

```diff
@@ -0,0 +1,292 @@
+from autoarray.plot.plotters import abstract_plotters
+from autoarray.plot.plotters import inversion_plotters
+from autoarray.plot.plotters import fit_imaging_plotters
+from autolens.plot.plotters import ray_tracing_plotters
+from autogalaxy.plot.mat_wrap import lensing_mat_plot, lensing_include, lensing_visuals
+from autolens.fit import fit as f
+
+import numpy as np
+
+
+class FitImagingPlotter(fit_imaging_plotters.FitImagingPlotter):
+    def __init__(
+        self,
+        fit: f.FitImaging,
+        mat_plot_2d: lensing_mat_plot.MatPlot2D = lensing_mat_plot.MatPlot2D(),
+        visuals_2d: lensing_visuals.Visuals2D = lensing_visuals.Visuals2D(),
+        include_2d: lensing_include.Include2D = lensing_include.Include2D(),
+    ):
+
+        super().__init__(
+            fit=fit,
+            mat_plot_2d=mat_plot_2d,
+            include_2d=include_2d,
+            visuals_2d=visuals_2d,
+        )
+
+    @property
+    def visuals_with_include_2d(self) -> "vis.Visuals2D":
+        """
+        Extracts from a `Structure` attributes that can be plotted and return them in a `Visuals` object.
+
+        Only attributes with `True` entries in the `Include` object are extracted for plotting.
+
+        From an `AbstractStructure` the following attributes can be extracted for plotting:
+
+        - origin: the (y,x) origin of the structure's coordinate system.
+        - mask: the mask of the structure.
+        - border: the border of the structure's mask.
+
+        Parameters
+        ----------
+        structure : abstract_structure.AbstractStructure
+            The structure whose attributes are extracted for plotting.
+
+        Returns
+        -------
+        vis.Visuals2D
+            The collection of attributes that can be plotted by a `Plotter2D` object.
+        """
+
+        visuals_2d = super(FitImagingPlotter, self).visuals_with_include_2d

```

Comment:
There's no need to pass arguments to super here, unless you're dealing with mixins